alter_michigan_databases.sh

```diff
@@ -15,26 +15,45 @@ copy_databases () {
   do
     mv "$f" "${db}_test${f#${db}}"
   done
-  cd ../..
+  cd -
 }
 
 alter_yelp () {
-  for f in "$DATABASE_DIR/yelp_test"/*.sqlite
+  for f in `ls $DATABASE_DIR/yelp_test/*.sqlite`
```

Comment:
Just curious, did the previous expression not work for you?

Reply:
I did not, for some reason.

Maybe you ran this locally using zsh, which is now the default on Macs?